.github/workflows/pre-commit.yml

```diff
@@ -34,5 +34,12 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
     - uses: actions/checkout@v3
+      with:
```

Comment:
This updates the pre-commit hooks to only run on files that were modified as part of this pull request.